packages/agent/src/openid4vc/registerDcApi.ts

```diff
@@ -105,15 +139,18 @@ export async function registerCredentialsForDcApi(agent: EitherAgent) {
         title: display.name,
         subtitle: `Issued by ${display.issuer.name}`,
         claims: mapSdJwtAttributesToClaimDisplay(sdJwtVc.prettyClaims),
+        iconDataUrl,
       },
     } as const
   })
 
-  const credentials = [...sdJwtCredentials, ...mdocCredentials]
+  const credentials = await Promise.all([...sdJwtCredentials, ...mdocCredentials])
   agent.config.logger.trace('Registering credentials for Digital Credentials API', {
     credentials,
   })
+
   await registerCredentials({
     credentials,
+    matcher: 'ubique',
```

Comment:
Testing?

Reply:
The library supports multiple matchers now and we use this one in our wallet, so not for testing